tests/test_tokenization_common.py

```diff
@@ -2977,8 +2977,8 @@ def test_padding(self, max_length=50):
                 input_r = tokenizer_r.encode_plus("This is a input 1")
                 input_r = tokenizer_r.pad(input_r)
 
-                input_p = tokenizer_r.encode_plus("This is a input 1")
-                input_p = tokenizer_r.pad(input_p)
+                input_p = tokenizer_p.encode_plus("This is a input 1")
+                input_p = tokenizer_p.pad(input_p)
```

Comment:
I think this is an overlooking when copying test cases.

Reply:
oh! Great catch! Thanks! :+1: